packages/theia-integration/package.json

```diff
@@ -17,15 +17,22 @@
     "glsp",
     "diagram editor"
   ],
-  "homepage": "https://www.eclipse.org/glsp",
-  "bugs": "https://github.com/eclipse-glsp/glsp-theia-integration/issues",
   "author": {
-    "name": "EclipseSource"
+    "name": "Eclipse GLSP"
   },
+  "homepage": "https://www.eclipse.org/glsp/",
   "repository": {
     "type": "git",
     "url": "https://github.com/eclipse-glsp/glsp-theia-integration.git"
   },
+  "bugs": "https://github.com/eclipse-glsp/glsp/issues",
+  "contributors": [
```

Comment:
I'm wondering whether we should either add all committers or no one here?

Reply:
Actually that's something I wanted to discuss in the next meeting. IMO it is quite tedious to maintain a list of all committers here. However, omitting it entirely might also be contra productive because it we probably need this list for sponsored and/or external contributions.
So I'd propose that we initially only list the package creator here. External contributors are free to extend this list.
Alternatively we could also use a generic author like:
```json
    "name": "Eclipse GLSP",  
    "email": "<someGenericMailAddress>",   
    "url": "https://www.eclipse.org/glsp/"
```
or 

```json
    "name": "EclipseSource",  
    "email": "<mailOfPackageCreator>",   
    "url": "https://eclipsesource.com/"
```

The same also has to be considered for the `<developers>` section in pom.xml files.

If we follow the Theia/Sprotty pattern of not maintaining contributor lists in the copyright header we then have the following possibilities to mention external contributors/sponsors

- The commit message
- The changelog
- The corresponding section in the package.json or pom.xml

Reply:
Ah ok I see, thanks! Let's discuss this in a separate issue then and not block this change.
Thanks!

Reply:
I created https://github.com/eclipse-glsp/glsp/issues/246 to track this